src/settings/validations/validators.js

```diff
@@ -1,8 +1,34 @@
 /* ---------------------------------------------------------------------------------------------
  *  All validators functions. A validator is for checking if an input is valid,
- *  consists of the relvant charecters.
+ *  consists of the relevant characters.
  *-------------------------------------------------------------------------------------------- */
 
+import { solAdditionalContracts, solidityObj } from '../stores/store'
+
+function contractValidators() {
```

Comment:
has to be separated! should not define function inside return. with significant function names